pkg/ovs/openflow/interfaces.go

```diff
@@ -78,7 +78,7 @@ type Bridge interface {
 	DumpTableStatus() []TableStatus
 	// DumpFlows queries the Openflow entries from OFSwitch. The filter of the query is Openflow cookieID; the result is
 	// a map from flow cookieID to FlowStates.
-	DumpFlows(cookieID, cookieMask uint64) map[uint64]*FlowStates
+	DumpFlows(cookieID, cookieMask uint64) (map[uint64]*FlowStates, error)
```

Comment:
any reason for this change? is it for consistency with the rest of the interface?